pkg/kubelet/pleg/generic.go

```diff
@@ -255,11 +260,6 @@ func (g *GenericPLEG) relist() {
 				needsReinspection[pid] = pod
```

Comment:
Thoughts on updating this variable name to be something like `needsReinspectionDuringNextRelisting`? When reading this code, it took me a while to realize that `needsReinspection` pods wouldn't actually be considered until the next relist.